app/src/main/java/com/yeonae/chamelezone/view/place/PlaceDetailActivity.kt

```diff
@@ -1,18 +1,37 @@
 package com.yeonae.chamelezone.view.place
 
 import android.os.Bundle
+import android.util.Log
+import android.view.View
+import android.view.ViewGroup
 import androidx.appcompat.app.AppCompatActivity
 import com.yeonae.chamelezone.R
 import com.yeonae.chamelezone.adapter.ImageViewPagerAdapter
 import com.yeonae.chamelezone.view.place.adapter.PlaceDetailPagerAdapter
 import kotlinx.android.synthetic.main.activity_place_detail.*
 
 class PlaceDetailActivity : AppCompatActivity() {
-    private val PLACE_NAME = "placeName"
-    private val PLACE_NUMBER = "placeNumber"
+
+
+    companion object{
+        private const val PLACE_NAME = "placeName"
+        private const val PLACE_NUMBER = "placeNumber"
+    }
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_place_detail)
+
+        vp.post {
```

Comment:
```suggestion
        vp_what.post {
```
뒤에 식별할 수 있는 네이밍을 추가해주세요.

Reply:
수정했습니다
d9bf9e8